src/components/DateRangePicker.jsx

```diff
@@ -361,6 +364,8 @@ export default class DateRangePicker extends React.Component {
           onNextMonthClick={onNextMonthClick}
           monthFormat={monthFormat}
           withPortal={withPortal || withFullScreenPortal}
+          hidden={!focusedInput}
```

Comment:
Can we change this name to `isVisible` instead of `hidden`? I think that is more in line with the other nomenclature in these files.

I think once we move to `react-with-styles`, we hopefully won't need this anyways. since the component will be unmounted/remounted every time.